jekyll/_posts/2020-10-20-one-web-framework-to-rule-over-all-fashions.md

```diff
@@ -87,7 +87,7 @@ app.run()
 
 The topic of today is to extend `Prologue` with the powerful macros outside the web framework.
 
-If you love `Flask` web framework, you may prefer the fashion below which uses `pragma` to simulate decorator.
+If you love `Flask` web framework, you may prefer the fashion below which uses `pragmas` to simulate decorator.
```

Comment:
I don't think there's need to quote this at all - you're not talking about some specific code, you're just talking about pragmas
```suggestion
If you love `Flask` web framework, you may prefer the fashion below which uses pragmas to simulate decorator.
```